cytoxnet/dataprep/io.py

```diff
@@ -3,27 +3,36 @@
 """
 
 import pandas as pd
+import os
 
 
-def load_data(datafile, columns='all'):
+def load_data(csv_file, col_id=None):
     """
     Loads a data file into a dataframe containing the raw data.
 
     Parameters
     ----------
     datafile: the file containing the data to be loaded
-    columns: the column (names?numbers?either?) that the user wants to load,
-        default='all'
+    col_id: the column (names?numbers?either?) that the user wants to
+        use to remove duplicates (ie remove any duplicates based on
+        the inputted column id), default=None
```

Comment:
Might be advantageous if the column named to `col_id` was explicitly supposed to contain the unique molecular identifiers.  I think this is already sort of implied in your description, but if we made it explicit, we could use the `col_id` input for both this function and as the id column for my splitting/transforming function.